packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -206,12 +212,19 @@ export const Webchat = forwardRef((props, ref) => {
       />
     </div>
   )
-
-  const persistentMenuOptions =
-    getThemeProperty('userInput.persistentMenu') || props.persistentMenu
+  const animationsEnabled = getThemeProperty('animations.enable', true)
+  const persistentMenuOptions = getThemeProperty(
```

Comment:
Here we lose `props.persistentMenu`?

Reply:
already in theme an getting in with getThemeProperty 👍